### PR TITLE
docs: add-ma-card-docs

### PR DESCRIPTION
**Pull request by alexcastelo95**
### Summary
Add emded docs for MaCard component from Figma. I've also added some basic tests. It is related to [this](https://holaluz.kanbanize.com/ctrl_board/65/cards/23083/details/) Kanbanize card.

### Resources
- [Link en Figma](https://www.figma.com/proto/7RhweKibaUnxX6WYBl7zKW/%E2%9A%A1%EF%B8%8F-Watt-Design-System?embed_host=share&hotspot-hints=0&kind=&node-id=5664%3A338&page-id=5314%3A1402&scaling=scale-down-width&viewport=1280%2C446%2C0.48540157079696655&hide-ui=1)
- [Docu en Notion](https://www.notion.so/holaluz/a1cb451d360941d98f08831cf679edab?v=f7024af195d34caface8649e58f98c12&p=c50116a66ba844b7b3398abcdb0088da)

### CODE REVIEW

**Review by edufarre — APPROVED** [review]
As we are targeting documentation, the title of the PR (the final commit message when doing squash and merge) should be `docs:` **not** `feat:`

File: docs/components/MaCard.docs.mdx [hunk 1]
```diff
@@ -1,3 +1,9 @@
-# Card
+import { Meta, Story } from '@storybook/addon-docs/blocks'
```
[the remark this answers is not in the document]
Comment (alexcastelo95) [reply docs/components/MaCard.docs.mdx:1]: You're right. Does this mean that I can delete the `MaCard.stories.js` file?

File: src/components/MaCard/MaCard.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      color: 'white',
+    })
+
+    expect(contentWrapper).toHaveClass('ma-card--white')
+  })
+})
+
+function renderComponent(props) {
+  const utils = render(MaCard, {
+    props: { ...props },
```
[the remark this answers is not in the document]
Comment (alexcastelo95) [reply src/components/MaCard/MaCard.spec.js:27]: I am not sure if I understand what you are saying. As the MaCard component does not have any identifier I have chosen to pass it a slot with a text inside to locate it.

I followed a similar approach to what was used in [MaColumns](https://github.com/holaluz/margarita/blob/9c5dc843f15b7f48a07b21ab71b80957ca437b7b/src/components/MaColumns/MaColumns.spec.js).

File: docs/components/MaCard.docs.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (edufarre) [reply docs/components/MaCard.docs.mdx:1]: Nope, one thing is the documentation, which can be written with `markdown` and another thing is the `story` per se, which allows to to interact with the desired component using Storybook Controls :)

File: src/components/MaCard/MaCard.spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (edufarre) [reply src/components/MaCard/MaCard.spec.js:27]: Just referenced the props functions' input variable, nothing to say about the slot 👯  If we define props as `{}` as default if none parameters are passed to the function, there is no need to do the disrupting in the `render` function

[the remark this answers is not in the document]
Comment (alexcastelo95) [reply src/components/MaCard/MaCard.spec.js:27]: Got it! Thanks for your clarification.

**Review by afontcu — APPROVED** [review]

**Pull request merged**